app/code/Magento/InventorySales/Plugin/CatalogInventory/StockManagement/ProcessRegisterProductsSalePlugin.php

```diff
@@ -99,20 +100,19 @@ public function aroundRegisterProductsSale(StockManagement $subject, callable $p
         if (empty($items)) {
             return [];
         }
+        $productSkus = $this->getSkusByProductIds->execute(array_keys($items));
+        $excludedItems = $this->getExcludedItems($items, $productSkus);
+        if (!empty($excludedItems)) {
+            $proceed($excludedItems, $websiteId);
```

Comment:
we no need to call the initial method in any case , under any condition.
That's our goal - to suppress one 

We want to place reservations even on Single Stock

Reply:
@maghamed, done. I've left only new behavior.